test/WebJobs.Script.Tests.Integration/CSharpEndToEndTests.cs

```diff
@@ -324,12 +324,12 @@ public CSharpEndToEndTests(TestFixture fixture) : base(fixture)
         {
             var request = new HttpRequestMessage
             {
-                RequestUri = new Uri($"http://localhost/api/HttpTriggerToBlob?suffix=TestSuffix"),
+                RequestUri = new Uri($"http://localhost/api/HttpTriggerToBlob?Suffix=TestSuffix"),
```

Comment:
Just making these Pascal cased for consistency - query/headers are case insensitive